tests/test_copy.py

```diff
@@ -1,28 +1,75 @@
 import copy
+
 import pytest
 
-from multidict._multidict import (MultiDict, CIMultiDict,
-                                  MultiDictProxy, CIMultiDictProxy)
-from multidict._multidict_py import (MultiDict as PyMultiDict,
+from multidict._compat import USE_CYTHON
+
+if USE_CYTHON:
+    from multidict._multidict import (MultiDict, CIMultiDict,
+                                      MultiDictProxy, CIMultiDictProxy)
+
+from multidict._multidict_py import (MultiDict as PyMultiDict,  # noqa: E402
                                      CIMultiDict as PyCIMultiDict,
                                      MultiDictProxy as PyMultiDictProxy,
                                      CIMultiDictProxy as PyCIMultiDictProxy)
 
 
-@pytest.fixture(params=[MultiDict, PyMultiDict,
-                        CIMultiDict, PyCIMultiDict],
-                ids=['MultiDict', 'PyMultiDict',
-                     'CIMultiDict', 'PyCIMultiDict'])
+@pytest.fixture(
+    params=(
```

Comment:
I prefer more dense formatting but can live with your notation

Reply:
It looked less maintainable and less readable to me when it was dense.